Features/NotesFeature/NotesViewModel.swift

```diff
@@ -66,10 +71,40 @@ final class NotesViewModel: ObservableObject {
         }
     }
 
+    func prepareNotesForSharing() async -> String {
+        do {
+            return try await crasher.recordError("Failed to share notes") {
+                var notesText = ""
+                for note in await notes {
+                    let noteText = if let noteContent = note.note.note, noteContent != "" {
+                        "Note: \(noteContent.trimmingCharacters(in: .newlines))\n"
```

Comment:
This needs to be localized. Maybe we can just not have "Note: " as a prefix and just show the text directly.